src/modules/i18n/locales/de.json

```diff
@@ -76,7 +76,7 @@
     "connect": "Verbinden",
     "contact_us": "Kontakt",
     "contextpopup_lv03_url": "https://www.swisstopo.admin.ch/de/landestriangulation-lv03",
-    "contextpopup_lv95_url": "https://www.swisstopo.admin.ch/de/landesvermessung-lv95",
+    "contextpopup_lv95_url": "https://www.swisstopo.admin.ch/de/schweizer-koordinatensystem",
```

Comment:
Those are Not correct url ans shouöd Not be in this PR

Reply:
@ltkum You seem to have changed that entry in the spreadsheet, should the value in table be changed back?